package.json

```diff
@@ -21,6 +21,9 @@
       ],
       "core": [
         "./build/core.d.ts"
+      ],
+      "internals": [
```

Comment:
Let’s not expose our internal stuff. Users tend to rely on internal, protected, hidden APIs and come upset on BC. 

I kbow this because got a lot of it in deployer.org

Reply:
Ok